modules/examples/example-tf/modulestack.yaml

```diff
@@ -0,0 +1,49 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: This template deploys a Module specific IAM permissions
+
+Parameters:
+  DeploymentName:
+    Type: String
+    Description: The name of the deployment
+  ModuleName:
+    Type: String
+    Description: The name of the Module
+  RoleName:
+    Type: String
+    Description: The name of the IAM Role
+
+Resources:
+  Policy:
+    Type: "AWS::IAM::Policy"
+    Properties:
+      PolicyDocument:
+        Statement:
+          # Terraform backend specific permissions
+          - Action:
+              - "s3:List*"
+            Effect: Allow
+            Resource: 
+              - "arn:aws:s3:::addf-tf*"
+          - Action:
+              - "s3:PutObject"
+              - "s3:DeleteObject"
+              - "s3:GetObject"
+            Effect: Allow
+            Resource: 
+              - !Sub "arn:aws:s3:::addf-tf*/${DeploymentName}/${ModuleName}*"
+          - Effect: Allow
+            Action:
+              - "dynamodb:DescribeTable"
+              - "dynamodb:GetItem"
+              - "dynamodb:PutItem"
+              - "dynamodb:DeleteItem"
+            Resource: 
+              - !Sub "arn:aws:dynamodb:${AWS::Region}:${AWS::AccountId}:table/addf-tf*"
```

Comment:
What creates the Dynamo and State bucket here? Manual for now?

Reply:
Manual for now, but i am working on a cdk stack which creates those resources (if someone wants to use them)